src/content/getting-started/prepare-your-provider-account/vmware-vsphere/_index.md

```diff
@@ -3,11 +3,103 @@ title: Prepare your provider account for VMware vSphere
+
+{{< tabs >}}
+{{< tab id="flags-kubevip" title="kube-vip">}}
+Since vSphere has no concept of load balancers out of the box, Cluster API ships with [kube-vip]({{< relref "/vintage/advanced/cluster-management/vsphere-kubevip" >}}), a layer two load balancer that works with [ARP](https://en.wikipedia.org/wiki/Address_Resolution_Protocol) requests. By default, `kube-vip` only handles the Kubernetes API access. Still, at Giant Swarm, we also deploy the `kube-vip` provider to offer the capability to create services of type load balancer.
+
+The ARP layer two protocol informs the network of the location of a new host address. `kube-vip` runs in-cluster as opposed to a more traditional external load-balancer that will forward IP packets to its upstream servers.
+
+![CAPV kubevip](capv-kubevip-excalidraw.png)
+
+Due to the in-cluster operation of `kube-vip`, the cluster network where this component is deployed must have a dedicated subnet range outside of the DHCP scope. To avoid IP conflicts, we recommend having one subnet per management cluster.
+
+![CAPV kubevip IPAM](capv-kubevip-ipam-excalidraw.png)
+
+When deploying a Cluster API cluster, it automatically selects an IP from the IP pool by default. However, to have available IPs for services of type load balancer in the workload cluster, you must explicitly set a CIDR in the nodes' subnet.
+
+Learn more about how to configure `kube-vip` in the [advanced documentation]({{< relref "/vintage/advanced/cluster-management/vsphere-kubevip" >}}).
+{{< /tab >}}
+
+{{< tab id="flags-nsxalb" title="NSX ALB">}}
+When using NSX Advanced Load Balancer (NSX ALB), [there are several components](https://docs.vmware.com/en/VMware-vSphere/7.0/vmware-vsphere-with-tanzu/GUID-A247F5F2-AC7E-48E7-B615-F8D361C7292A.html) involved to enable load balancer capabilities in Kubernetes.
+
+The `controller` in NSX ALB plays a pivotal role. It's responsible for communicating the operations requested to the vCenter Server, ensuring the smooth functioning of the load balancer. Additionally, there is a `service engine` to manage virtual IP addresses and a Kubernetes `operator` to reconcile the NSX ALB resources in the clusters.
+
+![CAPV kubevip](capv-nsxalb-excalidraw.png)
+
+{{< /tab >}}
+{{< /tabs >}}
+
+__Note__: The cluster nodes must have access to the vCenter endpoint on port 443 so that the controllers can manage the cluster's lifecycle.
+
+## Virtual machine templates
+
+To provision the virtual machines (VMs) for the cluster nodes, Giant Swarm needs permissions to upload `VM templates` to vCenter Server. The templates use a convention with the Linux distribution and Kubernetes version on the name (e.g., `flatcar-stable-3815.2.1-kube-v1.25.16`).
+
+<!-- Xavier here we dont have to mention anything about the VM sizing? -->
```

Comment:
@vxav do we need to mention anything about the VM sizing?

Reply:
No, sizing policies is only in cluster-cloud-director. In cluster-vsphere we specify the compute resources independently (ram, cpu...)

Reply:
So, does it mean those are provided in the cluster definition?